test/server/runner-test.js

```
@@ -325,4 +437,39 @@ describe('Runner', function () {
             testFilter(filter, expectedTestNames, done);
         });
     });
+
+    describe('.run()', function () {
+        it('Should raise exception if connection disconnected during tests', function (done) {
```

Should raise an exception if the connection breaks while tests are running